### DISCUSSION TITLE
请教：umi3.5想要实现404路由，如何配置

### DISCUSSION THREAD

**Discussion by jiajiaxing1** [umijs/umi#12411] [Q&A]
想要实现
点击/system-management/account可以展示页面
点击/system-management/account1跳转到404

在route里面配置了404路由 如图
<img width="499" alt="image" src="https://github.com/umijs/umi/assets/130677115/ebcfa746-5c42-487d-9b7f-3db5ec87528d">
最终生成的路由表是这样`[
  {
    "path": "/umi/plugin/openapi",
    "component": {}
  },
  {
    "path": "/",
    "component": {},
    "routes": [
      {
        "path": "/system-management",
        "name": "系统管理",
        "key": "系统管理",
        "icon": "control",
        "routes": [
          {
            "path": "/system-management/account",
            "name": "操作员管理",
            "key": "操作员管理",
            "icon": null
          },
          {
            "exact": true,
            "path": "/system-management",
            "redirect": "/system-management/account"
          }
        ],
        "children": [
          {
            "path": "/system-management/account",
            "name": "操作员管理",
            "key": "操作员管理",
            "icon": null
          },
          {
            "exact": true,
            "path": "/system-management",
            "redirect": "/system-management/account"
          }
        ]
      },
      {
        "path": "/*",
        "component": {},
        "layout": false,
        "name": "404",
        "exact": true
      }
    ],
    "children": [
      {
        "path": "/system-management",
        "name": "系统管理",
        "key": "系统管理",
        "icon": "control",
        "routes": [
          {
            "path": "/system-management/account",
            "name": "操作员管理",
            "key": "操作员管理",
            "icon": null
          },
          {
            "exact": true,
            "path": "/system-management",
            "redirect": "/system-management/account"
          }
        ],
        "children": [
          {
            "path": "/system-management/account",
            "name": "操作员管理",
            "key": "操作员管理",
            "icon": null
          },
          {
            "exact": true,
            "path": "/system-management",
            "redirect": "/system-management/account"
          }
        ]
      },
      {
        "path": "/*",
        "component": {},
        "layout": false,
        "name": "404",
        "exact": true
      }
    ]
  }
]`

**Comment by fz6m** [comment]
umi 3 不维护了，建议升级到 umi 4
